indico/modules/designer/templates/_list.html

```diff
@@ -109,13 +109,20 @@
                         {% if not clonable %}
                             title="{% trans %}This template cannot be cloned{% endtrans %}">
                         {% else %}
-                            title="{% trans %}Clone this template{% endtrans %}"
                             data-href="{{ url_for('.clone_template', target, template_id=tpl.id) }}"
-                            data-method="POST"
+                            title="{% trans %}Clone this template{% endtrans %}"
                             data-update="#template-list"
-                            data-confirm="{% trans title=tpl.title -%}
-                                              Are you sure you want to clone template '{{ title }}'?
-                                          {%- endtrans %}">
+                            {% if event %}
+                                data-method="POST"
+                                data-confirm="{% trans title=tpl.title -%}
+                                                Are you sure you want to clone template '{{ title }}'?
+                                            {%- endtrans %}">
```

Comment:
```suggestion
                                data-confirm="{% trans title=tpl.title -%}Are you sure you want to clone template '{{ title }}'?{%- endtrans %}">
```

Same below. This is a very long line but still looks much better than wrapping it.